build/nuget/uno.winui.common.targets

```diff
@@ -235,33 +231,6 @@
 		<Error Text="Building for the iOS Simulator requires the use of the static registrar. Make sure that `MtouchExtraArgs` contains `--registrar:static`. See https://github.com/unoplatform/uno/issues/9430 for more details." />
 	</Target>
 
-	<!-- Xamarin.iOS System.Memory version mismatch workaround -->
```

Comment:
These targets provide workarounds for Xamarin-specific build issues I think. So removing since Xamarin support is dropped.